packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -313,94 +309,88 @@ export class StorageService extends EventEmitter {
       this.logger.error('Error closing channels db', e)
     }
 
-    try {
-      await this.certificates?.close()
-    } catch (e) {
-      this.logger.error('Error closing certificates db', e)
-    }
+    // try {
+    //   await this.certificates?.close()
+    // } catch (e) {
+    //   this.logger.error('Error closing certificates db', e)
+    // }
+
     try {
       await this.certificatesRequests?.close()
     } catch (e) {
       this.logger.error('Error closing certificates db', e)
     }
+
     try {
       await this.communityMetadata?.close()
     } catch (e) {
       this.logger.error('Error closing community metadata db', e)
     }
+
     await this.__stopOrbitDb()
     await this.__stopIPFS()
   }
 
-  public async updatePeersList() {
-    const allUsers = this.getAllUsers()
-    const registeredUsers = this.getAllRegisteredUsers()
-    const peers = [...new Set(await getUsersAddresses(allUsers.concat(registeredUsers)))]
-    console.log('updatePeersList, peers count:', peers.length)
-    const community = await this.localDbService.get(LocalDBKeys.COMMUNITY)
-    this.emit(StorageEvents.UPDATE_PEERS_LIST, { communityId: community.id, peerList: peers })
-  }
-
-  public async loadAllCertificates() {
-    this.logger('Getting all certificates')
-    this.emit(StorageEvents.LOAD_CERTIFICATES, {
-      certificates: this.getAllEventLogEntries(this.certificates),
-    })
-  }
-
-  public async createDbForCertificates() {
-    this.logger('createDbForCertificates init')
-    this.certificates = await this.orbitDb.log<string>('certificates', {
-      replicate: false,
-      accessController: {
-        write: ['*'],
-      },
-    })
-    this.certificates.events.on('replicate.progress', async (_address, _hash, entry, _progress, _total) => {
-      const certificate = entry.payload.value
-
-      const parsedCertificate = parseCertificate(certificate)
-      const key = keyFromCertificate(parsedCertificate)
-
-      const username = getCertFieldValue(parsedCertificate, CertFieldsTypes.nickName)
```

Comment:
How does it relate to the certificates validation we're going to perform in certificate's store? Is it still something we need or can we ignore it while moving this part of code?

Reply:
See the comments above